### ISSUE TITLE
JSON output produces invalid json document

### ISSUE THREAD

**Issue by AndriyDmytrenko** [devops-kung-fu/bomber#230]
Hi!

When using scan with output=json it produces invalid json document:

```shell
$ bomber scan merged-bom.json --output=json > scan-report.json
$ file scan-report.json
scan-report.json: Unicode text, UTF-8 text, with very long lines (10321), with escape sequences
$ head scan-report.json
■ Scanning Files:
	merged-bom.json
{
	"meta": {
		"generator": "bomber",
		"url": "https://github.com/devops-kung-fu/bomber",
		"version": "0.5.0",
		"provider": "osv",
		"severityFilter": "",
		"date": "2024-08-21T16:37:59.927949+03:00"
$ 
```

<img width="563" alt="image" src="https://github.com/user-attachments/assets/404dfe6c-2da0-4f8a-9493-4651ed018dad">

These two lines at the beginning make it invalid.
So as a workaround I use command `sed '1,2d'` to skip them.

**Comment by djschleen** [comment]
I’ll check that out!

**Comment by djschleen** [comment]
Thanks for noticing that! This will be fixed in 0.5.1
